include/ua_server.h

```diff
@@ -1056,6 +1056,30 @@ UA_Server_deleteReference(UA_Server *server, const UA_NodeId sourceNodeId,
                           const UA_ExpandedNodeId targetNodeId,
                           UA_Boolean deleteBidirectional);
 
+/**
+ * .. _events:
+ *
+ * Events
+ * ------
+ * The method ``UA_Server_createEvent`` creates an event and represents it as node. The node receives a unique `EventId`
+ * which is automatically added to the node.
+ * The method returns a `NodeId` to the object node which represents the event. The `NodeId` can be used to set the
+ * attributes of the event.
+ *
+ * The method ``UA_Server_triggerEvent`` "triggers" an event by adding it to all monitored items of the specified
+ * origin node and those of all its parents. Any filters specified by the monitored items are automatically applied.
+ * Using this method deletes the node generated by ``UA_Server_createEvent``.
+ */
+#ifdef UA_ENABLE_SUBSCRIPTIONS_EVENTS
+
+UA_StatusCode UA_Server_createEvent(UA_Server *server, const UA_NodeId eventType,
```

Comment:
Comment if outnodeid can (can’t) be NULL.

Reply:
Outnodeid can be renamed to outeventid.

Is the generated nodeid always numeric? So we don’t have to clean up.

Reply:
It shouldn't be `outEventId` because EventID and NodeId are two different things. Maybe we should add both to the method header.

Every ObjectNode has of course a temporary nodeId which is valid until the event is triggered. Additionally every event has a variable with eventId which is automatically generated and identifies the event uniquely.

@aDogCalledSpot can you extend this in the documentation and add the eventId as a return parameter?

The generated nodeid is always numeric.